docs/concepts/pooling.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Ondat Storage Pooling"
+linkTitle: "Ondat Storage Pooling"
+weight: 1
+---
+
+## Overview
+
+> 💡 This feature is available in release `v2.9.0` or greater.
+
+Storage Pooling enables greater control over the type of storage a workload will use, this can be used to improve performance for storage-intensive applications and to reduce cost for less resource intensive applications.
+
+### What is Ondat Storage Pooling?
+
+The Storage Pooling feature allows creation of "Storage Pools", a Storage Pool is simply a group of drives connected to a set of machines.
```

Comment:
The Storage Pooling feature allows creation of "Storage Pools", **defined as** a group of drives connected to a set of machines.

Reply:
Should we format Storage Pool as `Storage Pool` ?

Reply:
> Should we format Storage Pool as `Storage Pool` ?

I don't think we should, the object the users interacts with is a `Pool`, however in the docs it makes more sense to refer to it a storage pool. Formatting it as `StoragePool` might confuse users into thinking that's the object they interact with